### PR TITLE
add dependabot.yml and update CHANGES.md

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
  - [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [x] CHANGES.rst has been updated (with summary of main changes)

### What kind of change does this PR introduce?

* Adds a dependabot configuration

### Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: daily
```
Comment (huard) [inline .github/dependabot.yml:11]:
```suggestion
      interval: weekly
```

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      time: '12:00'
+    open-pull-requests-limit: 10
+
+  - package-ecosystem: pip
+    directory: /
+    schedule:
+      interval: daily
```
Comment (huard) [inline .github/dependabot.yml:18]:
```suggestion
      interval: weekly
```

**Review by huard — APPROVED** [review]
Daily seems overkill, no ?

File: .github/dependabot.yml (hunk 1 above)
Comment (Zeitsperre) [reply .github/dependabot.yml:11]: Good point. We could push this to `monthly`.

**Pull request merged**